ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1606,6 +1606,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-sdn
+- as: azure-ipi-ultrassd-disks-p3-f28
```

Comment:
Shall we also add ci jobs for older versions, 4.10+ ?

Reply:
yes , I wanted to confirm for latest version if any issues with the job and tags , will copy them to earlier versions once review looks good.

Reply:
made the changes  , thanks for help @jianlinliu !!